video/out/vo_gpu_next.c

```diff
@@ -1131,6 +1131,14 @@ static bool draw_frame(struct vo *vo, struct vo_frame *frame)
                 pl_color_space_merge(&hint, source);
             if (target_unknown && !opts->target_trc && !pl_color_transfer_is_hdr(source->transfer))
                 hint = *source;
+            // Vulkan doesn't have support for gamma 2.2 transfer function,
+            // so even though requested preferred color space is gamma 2.2, we
+            // fallback to sRGB. sRGB itself is ambiguous, but at least we have
+            // options to control the behavior.
+            // TODO: Revise this after fix for linear transfers lands in libplacebo.
+            // <https://code.videolan.org/videolan/libplacebo/-/merge_requests/759>
+            if (hint.transfer == PL_COLOR_TRC_GAMMA22)
+                hint.transfer = PL_COLOR_TRC_SRGB;
```

Comment:
This might be a different bug, but bt709 content have hint.transfer GAMMA22. Not BT1886. So with this PR now, we'll decode bt709 with gamma22 instead of bt1886.